package.json

```diff
@@ -9,13 +9,14 @@
   "author": "Kaholo Team(Amit Alon)",
   "license": "ISC",
   "dependencies": {
-    "aws-eks-token": "^1.0.3",
-    "aws-sdk": "^2.1082.0",
-    "dayjs": "^1.10.4"
+    "aws-eks-token": "1.0.3",
+    "aws-sdk": "2.1082.0",
+    "dayjs": "1.10.4",
+    "kaholo-aws-library": "github:Kaholo/aws-plugin-library#v1.0.2"
```

Comment:
In other plugins we're using `kaholo-aws-plugin`, which emphasizes that there's a confusion about naming. Let's use `aws-plugin-library` to match the repo name.